source/nodejs/adaptivecards/src/carousel.ts

```diff
@@ -372,7 +424,15 @@ export class Carousel extends Container {
             keyboard: {
                 enabled: true,
                 onlyInViewport: true
-            }
+            },
```

Comment:
can you use [this pr](https://github.com/microsoft/AdaptiveCards/pull/7656) for the event handler?

Reply:
Yes, but I think we need an additional event if we use the navigation dots to switch between non-neighboring pages

Reply:
@anna-dingler, the pr has the pagenation (via 'dot') event.

Reply:
@jwoo-msft I refactored to use the event handlers from your PR. The timing is slightly more awkward because the swipe event was called more frequently.

I think using these event handlers will be good though if there is a better solution for updating peer location. I'm hoping to talk to @paulcam206 about this next week.